plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -616,7 +616,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                                               bool persist_until_expired,
                                               bool return_failure_trace,
                                               next_function<transaction_trace_ptr> next) {
-         bool exhausted = false;
+         bool success = false;
```

Comment:
I think something like `blk_exhausted` or `blk_not_exhausted` would be better. `success` is too vague in my opinion. It is not if the trx failed or succeeded, but rather if it ran out of net/time of the block.

Reply:
done